### PR TITLE
fix: disable LUIS directVersionPublish

### PR DESCRIPTION
**Pull request by liweitian**
## Description

Disable directVersionPublish, as it's causing compatible issue to previous users

Changes
* set directVersionPublish = false when publishing
* remove version property in publishing result(luis.settings.json) to avoid runtime picking direct version publishing api

## Task Item
#minor
## Screenshots
publish

### CODE REVIEW

File: Composer/packages/server/src/models/bot/builder.ts [hunk 1]
```diff
@@ -478,6 +483,48 @@ export class Builder {
     });
   }
 
+  private writeDialogAssets = async (contents, option: { force: boolean; out: string }) => {
```
Comment (boydc2014) [inline Composer/packages/server/src/models/bot/builder.ts:486]: rename this to write luis settings?

File: Composer/packages/server/src/models/bot/builder.ts [hunk 2]
```diff
@@ -444,13 +445,17 @@ export class Builder {
...
-      directVersionPublish: true,
-    });
-
-    await this.luBuilder.writeDialogAssets(buildResult, {
-      force: true,
-      out: this.generatedFolderPath,
+      directVersionPublish: false,
     });
+    // delete version property
+    for (let i = 0; i < buildResult.length; i++) {
```
Comment (boydc2014) [inline Composer/packages/server/src/models/bot/builder.ts:451]: add an if statement to make sure the id ends with json?

File: Composer/packages/server/src/models/bot/builder.ts (hunk 1 above)
Comment (boydc2014) [reply Composer/packages/server/src/models/bot/builder.ts:486]: I don't think you have to make `force` an option, keep it simple for now

**Comment by boydc2014** [comment]
@liweitian please fix the UT

**Comment by liweitian** [comment]
> @liweitian please fix the UT

fixed.

**Comment by boydc2014** [comment]
@luhan2017  can you help Tom signed off on this one

**Review by boydc2014 — APPROVED** [review]

**Pull request merged**